defaults/main.yml

```diff
@@ -116,6 +116,7 @@ galaxy_cvmfs_keys:
 
 galaxy_cvmfs_server_urls:
   - domain: galaxyproject.org
+    use_geoapi: true
```

Comment:
Given that this has not been enabled previously, I would suggest defaulting to the current value so as to not change the behaviour for current users?

Reply:
sounds reasonable.